docs/src/pages/components/SkipLink/WebSkipLinkPage.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import React from 'react';
+import BpkSkipLink from 'bpk-component-skip-link';
+import skipLinkReadme from 'bpk-component-skip-link/README.md';
+import { cssModules } from 'bpk-react-utils';
+
+import { WebComponentPage } from '../../../components/ComponentPage';
+import Paragraph from '../../../components/Paragraph';
+
+import STYLES from './skip-link-page.scss';
+
+const getClassName = cssModules(STYLES);
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default',
+    blurb: [
+      <Paragraph>
+        Skip links benefit sighted keyboard or switch users by providing them
+        with a way to skip over lengthy menus or filters. Skip links should be
+        absolute-positioned to ensure that when they become visible they do not
+        shift other content.
+        <br />
+        <br />
+        It’s ok for a skip link to overlap other content on the page, as long as
+        you are still able to tab forwards/backwards to that content. The skip
+        link is very noticeable, and doesn’t disappear immediately. These are
+        deliberate design decisions to ensure that a user tabbing quickly
+        through the page won’t miss the skip link.
+      </Paragraph>,
```

Comment:
I really like this description, in particular the part near the end explaining the deliberate design decision to pre-empt questions. I have a couple of small suggestions to improve it:

* Not a dealbreaker, but is it worth linking out to a canonical resource about skip links? MDN or Nomensa maybe? It'd be good to give people the opportunity to learn more.
* Can we mention something about this being a desktop-only pattern? I'm assuming it's a desktop-only pattern but I don't know for sure 😅

Reply:
I was actually gonna link to [webaim](https://webaim.org/techniques/skipnav/) but forgot to include it 🤦

Reply:
Nomensa have a good article too!
https://www.nomensa.com/blog/2004/what-are-skip-links

Reply:
btw skip links are also useful for keyboard/screen-reader users on mobile, so they apply equally.

Reply:
Oh cool, good to know.